### PR TITLE
add label support to search templates

### PR DESCRIPTION
**Pull request by aguilajesus**
with his change:
 - added "remote_template" label to differentiate  between local and imported templates.
 - added supported_os label (as seen in ForensicsArtifacts) to all imported templates.

### CODE REVIEW

File: tsctl [hunk 1]
```diff
@@ -521,10 +520,15 @@ class SearchTemplateManager(Command):
         if export_location:
             search_templates = []
             for search_template in SearchTemplate.query.all():
+                labels = []
+                for l in search_template.labels:
```
Comment (berggren) [inline tsctl]: use label instead of l, e.g:
```for label in ...```

File: tsctl [hunk 2]
```diff
@@ -534,23 +538,39 @@ class SearchTemplateManager(Command):
...
+                    continue
+
+                imported_template = SearchTemplate(
+                    name=name,
+                    user=User(None),
+                    query_string=query_string,
+                    query_dsl=query_dsl)
+
+                # Add supported_os labels.
+                for platform in search_template[u'supported_os']:
```
Comment (berggren) [inline tsctl]: to follow the naming, use:
```for supported_os in ...```

File: tsctl (hunk 1 above)
Comment (aguilajesus) [reply tsctl]: Done.

File: tsctl (hunk 2 above)
Comment (aguilajesus) [reply tsctl]: Done.

**Pull request merged**